src/app/onecx-concessions-ui.remote.module.ts

```diff
@@ -16,30 +16,59 @@ import {
   TranslateModule,
 } from '@ngx-translate/core';
 import {
-  addInitializeModuleGuard,
   AppStateService,
   ConfigurationService,
   createTranslateLoader,
   PortalCoreModule,
   PortalMissingTranslationHandler,
+  providePortalDialogService,
 } from '@onecx/portal-integration-angular';
-import { routes } from './app-routing.module';
 import { commonImports } from './app.module';
 import { metaReducers, reducers } from './app.reducers';
 import { Configuration } from './shared/generated';
 import { SharedModule } from './shared/shared.module';
 import { apiConfigProvider } from './shared/utils/apiConfigProvider.utils';
+import { AppEntrypointComponent } from './app-entrypoint.component';
+import { createAppEntrypoint, initializeRouter, startsWith } from '@onecx/angular-webcomponents';
+import { addInitializeModuleGuard } from '@onecx/angular-integration-interface';
+import { BrowserModule } from '@angular/platform-browser';
+import { DialogModule } from 'primeng/dialog';
 
 // Workaround for the following issue:
 // https://github.com/ngrx/platform/issues/3700
 const effectProvidersForWorkaround = [EffectsRunner, EffectSources, Actions];
 effectProvidersForWorkaround.forEach((p) => (p.ɵprov.providedIn = null));
 
+const routes: Routes = [
+  {
+    matcher: startsWith('travelconcession'),
+    loadChildren: () =>
+      import('./travelconcession/travelconcession.module').then(
+        (mod) => mod.TravelconcessionModule
+      ),
+  },
+  {
+    matcher: startsWith('traveloffering'),
+    loadChildren: () =>
+      import('./traveloffering/traveloffering.module').then(
+        (m) => m.TravelOfferingModule
+      ),
+  },
+  {
+    matcher: startsWith(''),
+    redirectTo: 'traveloffering',
+    pathMatch: 'full',
+  },
+];
+
 @NgModule({
   imports: [
+    BrowserModule,
```

Comment:
Can you add BrowserAnimationsModule just in case? We had an issue while migrating to angular 18 today without it. Thanks